pkg/secrets/annotations/annotation_parser.go

```diff
@@ -15,7 +16,14 @@ import (
 // by Kubernetes via the Downward API, based on Pod annotations that are defined
 // in a deployment manifest.
 func FromFile(path string) (map[string]string, error) {
-	annotationsFile, err := os.OpenFile(path, os.O_RDONLY, os.ModePerm)
+	// Use the standard OS
```

Comment:
@diverdane would you mind making these changes as part of this PR ? To maintain the convention in Go of explicitly calling out what the function is constructing e.g. `strings.NewReader`. 

FromFile -> NewAnnotationsFromFile
ParseReader -> NewAnnotationsFromReader

Reply:
Fixed.

Reply:
@doodlesbykumbi I actually changed this up a little bit.
Instead of using `NewAnnotationsFromFile`, I used `NewFromFile`. The reason I used this shortened form for the function name is that when it gets invoked from outside of this package, it will get invoked as:
```
    annotations.NewFromFile( . . . )
```
Whereas using the `NewAnnotationsFromFile` would stutter:
```
    annotations.NewAnnotationsFromFile( . . . )
```

I also used `newFromReader` instead of `NewAnnotationsFromReader` to be consistent with the `NewFromFile()`. I also made this a local (unexported) function, since it's only used internally.